arm-network/2016-09-01/swagger/network.json

```diff
@@ -6495,10 +6493,14 @@
           "description": "Gets or sets resource guid property of the LocalNetworkGateway resource"
         },
         "provisioningState": {
+          "readOnly": true,
           "type": "string",
           "description": "Gets provisioning state of the LocalNetworkGateway resource Updating/Deleting/Failed"
         }
       },
+      "required": [
+        "localNetworkAddressSpace"
+      ],
       "description": "LocalNetworkGateway properties"
     },
     "LocalNetworkGateway": {
```

Comment:
same reason.
"properties" property of "LocalNetworkGateway" should be required.